sympy/stats/crv_types.py

```diff
@@ -349,10 +354,8 @@ def Beta(name, alpha, beta):
     Parameters
     ==========
 
-    alpha : real positive
-        A shape
-    beta : real positive
-        A shape
+    alpha : Real number, `\alpha > 0`, a shape
```

Comment:
I don't think that these should be changed. According to [numpydoc specifications](https://numpydoc.readthedocs.io/en/latest/format.html), the parameter description is expected to be on another line.

Reply:
Oh ok. I will do that. @jksuom should I apply these changes for other cases as well?(when original is not according to the specifications) like https://github.com/sympy/sympy/blob/b884ca44c955fe8979b721a3fbe967cbaeaf0134/sympy/stats/crv_types.py#L190

Reply:
I have not included these changes in this PR

Reply:
I think you could change originals that are not to the specification.